verticapy/connection/utils.py

```diff
@@ -46,6 +46,16 @@ def get_connection_file() -> str:
     -------
     string
         the full path to the auto-connection file.
+
+    Example
+    -------
```

Comment:
Please add a line before to say that the connection file is the one used to store all the connections. 

.. warning:: Connections are stored in plain text format. In shared environments, it's recommended to create connections manually for added security.